src/README.md

```diff
@@ -0,0 +1,136 @@
+# API setup:
```

Comment:
Instead of calling this folder `src`, can we call it `server`? That way we can have `web-client` and `mobile-client` for the front end web and mobile code in the same repository without this being confusing.

Reply:
Sure thing